include/common/nas_headers.h

```diff
@@ -68,12 +72,13 @@ enum eps_nas_mesage_type {
 	DetachAccept = 0x46,
 	DetachRequest = 0x45,
 	TauAccept    = 0x49,
-    TauReject    = 0x4b,
+    	TauReject    = 0x4b,
 	ServiceReject = 0x4e,
 	AuthenticationRequest = 0x52,
-    IdentityRequest       = 0x55,
+    	IdentityRequest       = 0x55,
 	SecurityModeCommand = 0x5d,
-    EMMInformation = 0x61,
+    	EMMInformation = 0x61,
+	ActivateDedicatedBearerContextRequest = 0xc5,
```

Comment:
Why we need #define NAS_ACT_DED_BEARER_CTXT_ACPT, if we already have eps_nas_message_type ?

Reply:
In this enum, we are including outgoing NAS Messages (here, ActivateDedicatedBearerContextRequest).
In #define , we are including incoming NAS Messages (NAS_ACT_DED_BEARER_CTXT_ACPT & NAS_ACT_DED_BEARER_CTXT_RJCT).

Reply:
Well, I don't like it actually. Let me know if you agree to club them in single enum. I will do it, just want to know your opinion.

Reply:
Yes, we can club it . Changes done.